pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+junit_family=xunit2
```

Comment:
Can you please explain this line in comment or commit message?

Reply:
Updated the PR description